From: Hilko Bengen <bengen@debian.org>
Date: Thu, 15 Mar 2012 08:48:29 +0100
Subject: ruby: find files to install in correct directory

---
 ruby/Makefile.am |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ruby/Makefile.am b/ruby/Makefile.am
index efc2211..93a093e 100644
--- a/ruby/Makefile.am
+++ b/ruby/Makefile.am
@@ -53,7 +53,7 @@ RUBY_SITEARCH := $(shell $(RUBY) -rrbconfig -e "puts Config::CONFIG['sitearchdir
 install:
 	$(MKDIR_P) $(DESTDIR)$(RUBY_SITELIB)
 	$(MKDIR_P) $(DESTDIR)$(RUBY_SITEARCH)
-	$(INSTALL) -p -m 0644 lib/hivex.rb $(DESTDIR)$(RUBY_SITELIB)
-	$(INSTALL) -p -m 0755 ext/hivex/_hivex.so $(DESTDIR)$(RUBY_SITEARCH)
+	$(INSTALL) -p -m 0644 ${srcdir}/lib/hivex.rb $(DESTDIR)$(RUBY_SITELIB)
+	$(INSTALL) -p -m 0755 ${builddir}/ext/hivex/_hivex.so $(DESTDIR)$(RUBY_SITEARCH)
 
 endif
