From: Hilko Bengen <bengen@debian.org>
Date: Sat, 6 Aug 2011 16:34:09 +0200
Subject: python: don't build static library, don't build .so.0.0, symlinks
To: libguestfs@redhat.com

---
 python/Makefile.am |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/python/Makefile.am b/python/Makefile.am
index 8599863..89e66f8 100644
--- a/python/Makefile.am
+++ b/python/Makefile.am
@@ -36,6 +36,7 @@ libhivexmod_la_SOURCES = hivex-py.c
 libhivexmod_la_CFLAGS = -Wall -I$(PYTHON_INCLUDEDIR) \
 		        -I$(top_srcdir)/lib -I$(top_builddir)/lib
 libhivexmod_la_LIBADD = $(top_builddir)/lib/libhivex.la
+libhivexmod_la_LDFLAGS = -avoid-version -shared
 
 TESTS_ENVIRONMENT = \
 	PYTHONPATH=$(builddir):$(builddir)/.libs
-- 
