From: Hilko Bengen <bengen@debian.org>
Date: Thu, 15 Mar 2012 12:57:39 +0100
Subject: ruby: install to vendor directories

---
 ruby/Makefile.am | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/ruby/Makefile.am b/ruby/Makefile.am
index e78ab59..6d38fc1 100644
--- a/ruby/Makefile.am
+++ b/ruby/Makefile.am
@@ -45,13 +45,13 @@ all:
 	$(RAKE) build
 	$(RAKE) rdoc
 
-RUBY_SITELIB := $(shell $(RUBY) -rrbconfig -e "puts RbConfig::CONFIG['sitelibdir']")
-RUBY_SITEARCH := $(shell $(RUBY) -rrbconfig -e "puts RbConfig::CONFIG['sitearchdir']")
+RUBY_VENDORLIB := $(shell $(RUBY) -rrbconfig -e "puts RbConfig::CONFIG['vendorlibdir']")
+RUBY_VENDORARCH := $(shell $(RUBY) -rrbconfig -e "puts RbConfig::CONFIG['vendorarchdir']")
 
 install:
-	$(MKDIR_P) $(DESTDIR)$(RUBY_SITELIB)
-	$(MKDIR_P) $(DESTDIR)$(RUBY_SITEARCH)
-	$(INSTALL) -p -m 0644 ${srcdir}/lib/hivex.rb $(DESTDIR)$(RUBY_SITELIB)
-	$(INSTALL) -p -m 0755 ${builddir}/ext/hivex/_hivex.so $(DESTDIR)$(RUBY_SITEARCH)
+	$(MKDIR_P) $(DESTDIR)$(RUBY_VENDORLIB)
+	$(MKDIR_P) $(DESTDIR)$(RUBY_VENDORARCH)
+	$(INSTALL) -p -m 0644 ${srcdir}/lib/hivex.rb $(DESTDIR)$(RUBY_VENDORLIB)
+	$(INSTALL) -p -m 0755 ${builddir}/ext/hivex/_hivex.so $(DESTDIR)$(RUBY_VENDORARCH)
 
 endif
