From: Hilko Bengen <bengen@debian.org>
Date: Fri, 12 Aug 2011 13:17:06 +0200
Subject: configure: Apparently, python3's print requires parens.
To: libguestfs@redhat.com

---
 configure.ac |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6c38cb1..585ecb2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -237,8 +237,8 @@ PYTHON_PREFIX=
 PYTHON_VERSION=
 
 if test "x$PYTHON" != "xno"; then
-    PYTHON_PREFIX=`$PYTHON -c "import sys; print sys.prefix"`
-    PYTHON_VERSION=`$PYTHON -c "import sys; print sys.version[[0:3]]"`
+    PYTHON_PREFIX=`$PYTHON -c "import sys; print(sys.prefix)"`
+    PYTHON_VERSION=`$PYTHON -c "import sys; print(sys.version[[0:3]])"`
 
     AC_MSG_CHECKING([for Python include path])
     if test -z "$PYTHON_INCLUDEDIR"; then
-- 
