From: Hilko Bengen <bengen@debian.org>
Date: Mon, 16 Jul 2012 19:14:57 +0200
Subject: python bindings: Fix for Debian bug #681791

---
 configure.ac |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 1cfbc36..7b949b0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -293,8 +293,7 @@ if test "x$PYTHON" != "xno"; then
     dnl Look for libpython and some optional symbols in it.
     old_LIBS="$LIBS"
     if test "x$PYTHON_VERSION_MAJOR" = "x3"; then
-        dnl libpython3 is called "libpython3.Xmu.so"
-        LIBPYTHON="python${PYTHON_VERSION}mu"
+        LIBPYTHON="python${PYTHON_VERSION}"`python${PYTHON_VERSION}-config --abiflags`
     else
         LIBPYTHON="python$PYTHON_VERSION"
     fi
