From: Hilko Bengen <bengen@debian.org>
Date: Fri, 12 Aug 2011 12:03:13 +0200
Subject: run autoconf, automake
To: libguestfs@redhat.com

---
 configure          |    4 ++--
 python/Makefile.in |    5 +++--
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/configure b/configure
index 9645887..9fdc4aa 100755
--- a/configure
+++ b/configure
@@ -25260,8 +25260,8 @@ PYTHON_PREFIX=
 PYTHON_VERSION=
 
 if test "x$PYTHON" != "xno"; then
-    PYTHON_PREFIX=`$PYTHON -c "import sys; print sys.prefix"`
-    PYTHON_VERSION=`$PYTHON -c "import sys; print sys.version[0:3]"`
+    PYTHON_PREFIX=`$PYTHON -c "import sys; print(sys.prefix)"`
+    PYTHON_VERSION=`$PYTHON -c "import sys; print(sys.version[0:3])"`
 
     { $as_echo "$as_me:${as_lineno-$LINENO}: checking for Python include path" >&5
 $as_echo_n "checking for Python include path... " >&6; }
diff --git a/python/Makefile.in b/python/Makefile.in
index b8a60a0..6ce4bcd 100644
--- a/python/Makefile.in
+++ b/python/Makefile.in
@@ -142,8 +142,8 @@ am__v_lt_ = $(am__v_lt_$(AM_DEFAULT_VERBOSITY))
 am__v_lt_0 = --silent
 libhivexmod_la_LINK = $(LIBTOOL) $(AM_V_lt) --tag=CC \
 	$(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=link $(CCLD) \
-	$(libhivexmod_la_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o \
-	$@
+	$(libhivexmod_la_CFLAGS) $(CFLAGS) $(libhivexmod_la_LDFLAGS) \
+	$(LDFLAGS) -o $@
 @HAVE_PYTHON_TRUE@am_libhivexmod_la_rpath = -rpath $(pythondir)
 DEFAULT_INCLUDES = -I.@am__isrc@ -I$(top_builddir)
 depcomp = $(SHELL) $(top_srcdir)/build-aux/depcomp
@@ -939,6 +939,7 @@ EXTRA_DIST = \
 @HAVE_PYTHON_TRUE@		        -I$(top_srcdir)/lib -I$(top_builddir)/lib
 
 @HAVE_PYTHON_TRUE@libhivexmod_la_LIBADD = $(top_builddir)/lib/libhivex.la
+@HAVE_PYTHON_TRUE@libhivexmod_la_LDFLAGS = -avoid-version -shared
 @HAVE_PYTHON_TRUE@TESTS_ENVIRONMENT = \
 @HAVE_PYTHON_TRUE@	PYTHONPATH=$(builddir):$(builddir)/.libs
 
-- 
